src/Microsoft.Health.Fhir.Core/Features/Persistence/RawResource.cs

```diff
@@ -26,10 +28,16 @@ protected RawResource()
         }
 
         [JsonProperty("data")]
-        public string Data { get; protected set; }
+        public string Data { get; set; }
 
         [JsonProperty("format")]
         [JsonConverter(typeof(StringEnumConverter))]
         public FhirResourceFormat Format { get; protected set; }
+
+        [JsonProperty("versionSet")]
+        public bool VersionSet { get; protected set; }
+
+        [JsonProperty("lastUpdatedSet")]
+        public bool LastUpdatedSet { get; protected set; }
```

Comment:
If these flags will always have same value (either both true or both false), should we merge these into one?